docs/sphinx_api_docs_source/public_api_excludes.py

```diff
@@ -827,4 +827,19 @@
         name="get_batch_parameters_keys",
         filepath=pathlib.Path("great_expectations/datasource/fluent/sql_datasource.py"),
     ),
+    IncludeExcludeDefinition(
```

Comment:
If we were to remove the unsupported actions would we have to undo this change and remove these?

Reply:
If so, could you add it as a task to do in CORE-600?

Reply:
> If we were to remove the unsupported actions would we have to undo this change and remove these?

I assume yes, but I'm not 100% confident in my understanding of how the api docs get built

> If so, could you add it as a task to do in CORE-600?

yes, will do!

Reply:
@klavavej the api docs build is complex. I'm happy to chat about it.